spec/setup_test_database.rb

```diff
@@ -0,0 +1,18 @@
+# in spec/setup_test_database.rb
+
+require 'pg'
+
+def setup_test_database 
+  p "Setting up test database..."
+
+  connection = PG.connect(dbname: 'chitter_test')
+
+  # Clear the bookmarks table
+  connection.exec("TRUNCATE peeps;")
+end
+
+def persisted_data(id:)
```

Comment:
does this method duplicated your database_helper.rb?
perhaps you could refactor to a single method/helper